packages/cli/package.json

```diff
@@ -48,7 +48,9 @@
     "@vercel/node": "1.12.2-canary.4",
     "@vercel/python": "2.0.6-canary.4",
     "@vercel/ruby": "1.2.8-canary.3",
-    "update-notifier": "4.1.0"
+    "update-notifier": "4.1.0",
+    "vercel-plugin-middleware": "0.0.0-canary.5",
+    "vercel-plugin-node": "1.12.2-plugin.6"
```

Comment:
Why is this version so strange? Usually we use `-canary` suffix?

Reply:
Its also 81 MB which seems wrong 

https://packagephobia.com/result?p=vercel-plugin-node@1.12.2-plugin.6

Reply:
That's a good question, looks like it was originally done here:
https://github.com/vercel/vercel/pull/6861

@jaredpalmer

Reply:
I've changed it back to `canary` in the plugin-node PR, so next publish will use that tag.

Reply:
@TooTallNate What about the 81MB? Seems really large, doesn't seem right 🤔

Reply:
Oh I guess its 69.7MB now https://packagephobia.com/result?p=vercel-plugin-node@1.12.2-canary.4

That's basically the same as `@vercel/node` https://packagephobia.com/result?p=@vercel/node@1.12.1

So we can revisit that in a different PR 👍